<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteraciones</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aprendizajes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Error permisible</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="error_permisible">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="iteraciones">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aprendizaje">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>137</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="grafica_errores">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Grafica del historial del error (algebra)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generar_grafica_aprendizajes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Grafica de errorres con multiples aprendizajes (algebra)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grafica_comparativa">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar grafica Comparativa de YCs</string>
    </property>
   </widget>
   <widget class="QLabel" name="estado">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Para esta funcion no es necesario ingresar valores</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Para esta funcion no es necesario ingresar valores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grafica_errores_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Grafica del historial del error (tensor)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generar_grafica_aprendizajes_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Grafica de errorres con multiples aprendizajes (tensor)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Para esta funcion no es necesario ingresar valores</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
